pulser/simulation/simulation.py

```diff
@@ -75,6 +71,9 @@ def __init__(self, sequence: Sequence, sampling_rate: float = 1.0,
         self._tot_duration = max(
             [self._seq._last(ch).tf for ch in self._seq._schedule]
         )
+        self._interaction = 'ising'
+        if self._seq._in_xy:
+            self._interaction = 'XY'
```

Comment:
You can put this neatly in a ternary statement:
```suggestion
        self._interaction = 'XY' if self._seq._in_xy else 'ising'
```